src/main/java/team/creative/littletiles/client/tool/LittleToolPlacer.java

```diff
@@ -173,7 +173,9 @@ public void tick(Level level, Player player, @Nullable BlockHitResult blockHit)
         var mode = placer.getPlacementMode(stack);
         var matrix = placer.getMatrix(stack);
         var hasTiles = placer.hasTiles(stack);
-        var hash = stack.get(LittleTilesRegistry.DATA).hashCode();
+        var data = stack.get(LittleTilesRegistry.DATA);
+        if (data == null) return;
```

Comment:
Oh totally forgot about it, instead of returning I would rather set the hash to zero. Because the code below can be relevant.